### ISSUE TITLE
BUG: Client crash when calling SDL_GL_DeleteContext (vid_restart, closing ezquake)

### ISSUE THREAD

**Issue by meag** [ezQuake/ezquake-source#120]
Client can crash when calling SDL_GL_DeleteContext()
The passed parameter is correct, SDL calls wglDeleteContext(), and the access violation is in nvoglv32.dll
Internet searches indicate that this can happen if textures/buffers/gl-objects are not freed before calling wglDeleteContext.

Struggling to find exact steps to reproduce.  Also reported by locust.

**Comment by jite** [comment]
Hopefully fixed with ec90f2d420b2d7643927a956ff351d83f692b08d

**Comment by meag** [comment]
Report from Verro that he could reliably (4/4 times) make this happen in 3.0.1 by:

1) connecting to qtv
2) toggling gl_picmip
3) vid_restart

On discussion, we found that he had removed all external textures from his qw/ directory.  He tried adding 24-bit textures back in and couldn't replicate.  I tried again and found that it seemed more likely to happen if I:

1) Removed all external textures from /qw/
2) Connecting to qtv when the game was in progress (this may just be a symptom of the graphics system being busier with particles, enemy skins etc)
3) /vid_restart twice

However this doesn't always reproduce the problem.
